UHPostalService/UHPostalService/Pages/Tracking/Edit.cshtml

```diff
@@ -54,7 +54,7 @@
                 <span asp-validation-for="TrackingRecord.Destination" class="text-danger"></span>
             </div>-->
             <div>
-                <a asp-page="/Addresses/Edit" asp-route-id="@Model.TrackingRecord">Edit Destination</a>
+                <a asp-page="/Tracking/Checkout" asp-route-trnum="@Model.Package.Id">Edit Destination</a>
```

Comment:
these buttons on the edit pages need to be removed. I didnt know you were talking about doing this when you asked me